### PR TITLE
feat(client): new UI for updating user metadata

### PR DESCRIPTION
**Pull request by wreality**
This PR:
* Adds UI for updating profile metadata
* Adds validation on front end and backend for profile metadata fields
* Rearranges header dropdown links
* Removes account/profile from main left sidebar
* Updates account page to use UI components from this PR for consistency

### CODE REVIEW

**Comment by wreality** [comment]
This is suffering from a regression in the cypress tests for the user-search component and a contrast A11y error (due to an element fading in).

Review can probably start @gmeben and/or @Sruthin86  

And if you have any suggestions for the two cypress issues, I'm all ears.

**Review by gmeben — APPROVED** [review]
I didn't do an exhaustive review but this looks and feels really good in the browser. No test errors locally besides the usual nondeterministic stuff.

File: client/src/components/forms/ProfileMetadataForm.vue [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+          prefix="https://fb.com/"
+          class="col-md-6 col-12"
+          clearable
+        >
+          <template #prepend>
+            <q-icon
+              name="fab fa-facebook"
+              :class="{
+                'brand-active': v$.social_media.facebook.$model.length,
+              }"
```
[the remark this answers is not in the document]
Comment (wreality) [reply client/src/components/forms/ProfileMetadataForm.vue:43]: Fix in latest.  Might need restart / rebuild to load the icon library.

File: client/src/pages/Account/MetadataPage.vue [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<template>
+  <profile-metadata-form
+    ref="form"
+    :profile-metadata="profileMetadata"
```
[the remark this answers is not in the document]
Comment (wreality) [reply client/src/pages/Account/MetadataPage.vue:4]: Annoyingly its not the no profile metadata thats the problem.  Its that the query result returns null briefly even if it has a default empty object set.

[the remark this answers is not in the document]
Comment (wreality) [reply client/src/pages/Account/MetadataPage.vue:4]: Fixed in latest

**Pull request merged**